tools/link.py

```diff
@@ -2325,23 +2325,15 @@ def modularize():
   if async_emit != '' and settings.EXPORT_NAME == 'config':
     diagnostics.warning('emcc', 'EXPORT_NAME should not be named "config" when targeting Safari')
 
-  src = '''
-%(maybe_async)sfunction(moduleArg = {}) {
-
-%(src)s
-
-  return %(return_value)s
-}
-''' % {
-    'maybe_async': async_emit,
-    'src': src,
-    'return_value': return_value,
-  }
-
   if settings.MINIMAL_RUNTIME and not settings.PTHREADS:
     # Single threaded MINIMAL_RUNTIME programs do not need access to
     # document.currentScript, so a simple export declaration is enough.
-    src = '/** @nocollapse */ var %s = %s' % (settings.EXPORT_NAME, src)
+    src = f'''\
```

Comment:
I'm not sure why @nocollapse was added here.  That seems to only apply to object or to object properties, not to function.

Reply:
https://github.com/google/closure-compiler/wiki/Annotating-JavaScript-for-the-Closure-Compiler#nocollapse